articles/112_generated_interfaces_cpp.md

```diff
@@ -28,16 +30,25 @@ This article specifies the generated C++ code for ROS interface types defined in
 ## Namespacing
 
 All code of a ROS package should be defined in a namespace named after the package.
-To separate the generated code from other code within the package it is defined in a sub namespace:
+To separate the generated code from other code within the package it is defined in a sub-namespace:
+
 * namespace for ROS messages: `package_name::msg`.
 * namespace for ROS services: `package_name::srv`.
 
+<div class="alert alert-warning" markdown="1">
+  **Note:** Those namespaces are not consistent with ROS 1 usage.
+</div>
+
 
 ## Generated files
 
 Following the C++ style guide of ROS 2 the namespace hierarchy is mapped to a folder structure.
 The filenames use lowercase alphanumeric characters with underscores for separating words and end with either `.hpp` or `.cpp`.
 
+<div class="alert alert-warning" markdown="1">
+  **Note:** The suggested naming is not consistent with ROS 1 usage.
+</div>
+
```

Comment:
With your proposed change how can ROS 1 and ROS 2 header be included at the same time?

How will the C equivalent of the message header be named then?